orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -591,8 +583,8 @@ def __init__(self, parent=None):
         self.scroll.setHorizontalScrollBarPolicy(Qt.ScrollBarAlwaysOff)
         self.scroll.setVerticalScrollBarPolicy(Qt.ScrollBarAlwaysOn)
         self.scroll.resize(frame_layout.sizeHint())
-        self.scroll.setMinimumWidth(frame_layout.sizeHint().width() + 20)  # + scroll bar
```

Comment:
One tiny problem with this ... If I open Preprocess widget, without any input, and enable POS Tagger this is what I see:
![screen shot 2016-11-30 at 18 00 59](https://cloud.githubusercontent.com/assets/713026/20762562/0ec5011e-b727-11e6-8007-43b2c7f3dc5a.png)

We should widen the widget immediately after the section is enabled, i.e. we should not wait till the apply is called.